### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/SpectrumWorx#150]
Implement accessibility across the plugin.

For the preset browser, make sure Backspace and left arrow keys navigate to one level above, and Enter and right arrow keys enter into the folder.

**Comment by mkruselj** [comment]
Also think we should have keybinds for direct focus/switching to presets (Alt+P), switching between User/Factory (Alt+U), switching to FX parameters (Alt+F), switching between middle module parameters and currently selected parameter (Alt+M), engine settings (Alt+W), GUI settings (Alt+G), About (F12).
